modules/shared/components/molecules/Misc/Misc.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import className from 'classnames';
+import type { FC, ReactElement } from 'react';
+import ErrorIcon from '../../icons/miscError.svg';
+import SuccessIcon from '../../icons/miscSuccess.svg';
+import styles from './Misc.module.css';
+import type { IMisc } from './types/IMisc';
+import * as EMisc from './types/EMisc';
+
+const Misc: FC<IMisc.IProps> = ({ type, msg, subMsg }): ReactElement => {
+  const classes: string = className({
```

Comment:
change the name _classes_ to a more descriptive name like _box-wrapper_ for example